dnf5/main.cpp

```diff
@@ -713,6 +713,14 @@ int main(int argc, char * argv[]) try {
 
         context.apply_repository_setopts();
 
+        for (const auto & tsflag : base.get_config().tsflags().get_value()) {
+            if (tsflag == "test") {
+                std::cout << "Test mode enabled: Only package downloads, gpg key installations and transaction checks "
+                             "will be performed."
+                          << std::endl;
+            }
+        }
+
```

Comment:
I'm not entirely sure of the placement of this message. Seems to me this will actually be printing for any command regardless of whether it performs a transaction, if the `"test"` flag is enabled? Previously this was only happening in userconfirm - which arguably is not the entirely best place, as the confirmation can be for something else than a transaction?

I think this should go after printing the transaction table in the `if (auto goal = context.get_goal(false)) {` condition block. @mblaha @jrohel agreed?

Reply:
I concur. Also printing it after the transaction table gives the message better visibility (especially for large transactions).